### ISSUE TITLE
Nyc code coverage 

### ISSUE THREAD

**Issue by TravisDev** [adamgruber/mochawesome#271]
I currently use a string of something like this to run my tests and generate my reports:

    "test": "nyc --reporter=html --reporter=text mocha  --reporter mochawesome --reporter-options reportDir=report,reportFilename=index.html dist/test/*.js 2>&1",

It would be absolutely AMAZING if we could use mochawesome  to integrate the coverage report into the mocha report.

My only use case at the moment is to have a seperate report as an alternative.

**Comment by apgapg** [comment]
Yes code coverage is missing in index.html website. its good if integrated

**Comment by adamgruber** [comment]
Coverage reporting is out of scope for mochawesome, which reports test *results* — coverage is produced separately by tools like `nyc`/`c8`, each with their own report output. Combining a coverage report into the mochawesome HTML isn't planned. Closing as out of scope, though the idea is noted for possible future consideration.

---
_Triaged and posted via Claude Code, on behalf of @adamgruber._
